qa/covid19_qa/elasticsearch_qa.py

```diff
@@ -1,27 +1,55 @@
+#!/usr/bin/python
+# -*- coding: latin-1 -*-
```

Comment:
I think it's more standard to use utf-8:

```suggestion
# -*- coding: utf-8 -*-
```

Is the file really encoded in latin-1? (you can check with `file qa/covid19_qa/elasticsearch_qa.py`).